astro.config.mjs

```diff
@@ -37,6 +37,9 @@ export default defineConfig({
     }, {
       label: 'How Treblle works',
       link: '/how'
+    }, {
+      label: 'Aspen',
+      link: '/aspen/'
```

Comment:
I would add Aspen last, ie after Security as it is a Treblle product.

Reply:
Updated.